eng/build.yml

```diff
@@ -412,8 +411,11 @@ jobs:
           displayName: Run dotnet new Integration Tests
 
       - ${{ if not(contains(parameters.agentOs, 'Windows_NT')) }}:
-        - script: eng/common/cibuild.sh
+        - script: eng/common/build.sh
                     --configuration $(_BuildConfig)
+                    --restore
+                    --build
+                    --ci
```

Comment:
Do we want to run the unittests as well (`--test`)?

Reply:
Oh - I see tests are disabled in this step anyways

Reply:
yes, build step disabled tests.